tools/bridge/bridge/main.py

```diff
@@ -212,17 +230,15 @@ def main(config_path: str) -> None:
 
     pool = gevent.pool.Pool()
 
-    def raise_keyboard_interrupt():
-        raise KeyboardInterrupt()
-
     try:
         for coroutine_and_args in coroutines_and_args:
             pool.spawn(*coroutine_and_args)
 
-        gevent.signal(signal.SIGINT, raise_keyboard_interrupt)
-        gevent.signal(signal.SIGTERM, raise_keyboard_interrupt)
+        stop_pool = partial(stop, pool, APPLICATION_CLEANUP_TIMEOUT)
+        gevent.signal(signal.SIGINT, stop_pool)
```

Comment:
You could just replace that with 
```
for signum in (signal.SIGINT, signal.SIGTERM):
    gevent.signal(signum, stop, pool, APPLICATION_CLEANUP_TIMEOUT)
```

But I'm also fine with keeping it the way it is.